sled-agent/src/rack_setup/plan/service.rs

```diff
@@ -628,6 +637,47 @@ impl Plan {
             });
         }
 
+        // Provision Clickhouse server zones, continuing to stripe across sleds.
+        // TODO(https://github.com/oxidecomputer/omicron/issues/732): Remove
+        // Temporary linter rule until replicated Clickhouse is enabled
+        #[allow(clippy::reversed_empty_ranges)]
+        for _ in 0..CLICKHOUSE_SERVER_COUNT {
+            let sled = {
+                let which_sled =
+                    sled_allocator.next().ok_or(PlanError::NotEnoughSleds)?;
+                &mut sled_info[which_sled]
+            };
+            let id = OmicronZoneUuid::new_v4();
+            let ip = sled.addr_alloc.next().expect("Not enough addrs");
+            // TODO: Should this be a different port in case we want to have single node
+            // and replicated running side by side for some weird reason?
```

Comment:
This would not happen if we use the types I mention above. You can only have one kind of deployment in that case, which I think is probably what we want.

Reply:
Same reason as above, but this comment can be improved. I'll rephrase.